workers/unity/Packages/com.improbable.gdk.transformsynchronization/Systems/InterpolateTransformSystem.cs

```diff
@@ -100,23 +100,25 @@ protected override void OnUpdate()
                     var transformToInterpolateTo = ToBufferedTransform(lastTransformApplied);
 
                     var transformToInterpolateFrom = transformBuffer[transformBuffer.Length - 1];
-                    uint lastTickId = transformToInterpolateFrom.PhysicsTick;
+                    var lastTickId = transformToInterpolateFrom.PhysicsTick;
 
-                    uint remoteTickDifference = transformToInterpolateTo.PhysicsTick - lastTickId;
-                    if (remoteTickDifference == 0)
+                    if (transformToInterpolateTo.PhysicsTick <= lastTickId)
                     {
                         continue;
                     }
 
-                    uint ticksToFill =
-                        math.max((uint) (transformToInterpolateTo.PhysicsTick - lastTickId), 1);
-                    for (uint j = 0; j < ticksToFill - 1; ++j)
+                    var remoteTickDifference = (int) (transformToInterpolateTo.PhysicsTick - lastTickId);
```

Comment:
What are the types of `PhysicsTick` and `lastTickId`? 

If they are both `uint`s can this still cause a problem if `PhysicsTick` > `Int.MaxPositive` ?

Reply:
Verified that this will be fine